common-features/input-validation.md

```diff
@@ -11,6 +11,13 @@ position: 2
 
 The UI for Blazor suite supports and integrates seamlessly into Blazor's Forms and Validation infrastructure. All Telerik UI for Blazor Input components work out of the box when placed inside an `EditForm`, respond to `EditContext` changes and provide default invalid styles.
 
+In this article:
+
+* [Validation Basics](#validation-basics)
+* [Validation Mode for Simple Inputs](#validation-mode-for-simple-inputs)
```

Comment:
Consider making this plural - modes. This will encourage the reader to check the available options and reduce wondering what a validation mode is by itself.

Reply:
Updated.